Spider_scripts/aruodas_scraper.py

```diff
@@ -0,0 +1,31 @@
+import scrapy
+
+class aruodascrawl(scrapy.Spider):
+    name='aruodas'
+    start_urls=['https://m.en.aruodas.lt/butai/kaune/puslapis/1/?obj=1&FRegion=43&FDistrict=6']
+
+    def parse(self,response):
+        for flats in response.css('a.result-item-info-container-v3'):
+            region_street=flats.css('span.item-address-v3::text').get().strip().split(',')
+            all_info=flats.css('span.item-description-v3::text').get()
+            s1 = all_info.find('rooms')
+            s2 = all_info.find('m²')
+            s3 = all_info.find('floor')
+            s4 = all_info.find('year')
+            yield {
+                #'Pavadinimas': flats.css('span.item-address-v3::text').get().strip(),
+                'Rajonas':  region_street[0],
+                'Gatve' :  region_street[1],
+                'Kaina': flats.css('span.item-price-main-v3').xpath('span/text()').get().replace('€',''),
+                #'Visa_info': all_info,
+                'Dydis': all_info[(s1+6):(s2)].replace(',','.'), 
```

Comment:
when scraping, as much as possible this sort of code should be avoided. Using objects is always more stable than analysing strings. On this page there is no way to do it without, but by clicking you on items you could and you get more info. If you don't need more info, probably what you did is best